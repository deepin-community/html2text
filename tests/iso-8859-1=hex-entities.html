<html xmlns="http://www.w3.org/1999/xhtml">
	<head>
	</head>
	<body>
		<table>
			<tr>
				<td>&nbsp;</td>
			</tr>
			<tr>
				<td><a href="">&copy;  foo</a>
					&nbsp; &#8226; &nbsp;
					<a href="">bar</a>
				</td>
			</tr>
		</table>
	</body>
</html>
