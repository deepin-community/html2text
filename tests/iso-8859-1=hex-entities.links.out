iso-8859-1=hex-entities.default.out